packages/react-core/src/components/ActionList/ActionList.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/ActionList/action-list';
+
+export interface ActionListProps extends React.HTMLProps<HTMLDivElement> {
+  /** Children of the action list */
+  children?: React.ReactNode;
+  /** Flag indicating the action list contains icons */
+  hasIcons?: boolean;
```

Comment:
@mcoker I have a question.  Should the `pf-m-icons` modifier be applied whenever there is an icon in the Action list?  For instance the first example in the docs has the dropdown with kebab toggle (an icon).  Should the modifier be applied there?

Reply:
It's for an action list and/or group that has multiple plain buttons side by side where you want to remove any space between them. The modifier just removes space between items.

We have the same kind of modifier in the toolbar: https://github.com/patternfly/patternfly-react/blob/0e7bac0b9c897522bb93943e4606f5990fd41b8b/packages/react-core/src/components/Toolbar/ToolbarGroup.tsx#L16

Reply:
Should I update the prop description to better reflect this?

`Flag indicating the action list contains multiple icons and item padding should be removed`?

Reply:
yeah though I would say something more along the lines of "the action list or group contains a list of plain actions and the space between them is removed".